win32/ctags_vs2017.sln

```diff
@@ -0,0 +1,20 @@
+﻿
```

Comment:
I don't think adding "ctags_vs2017.sln" is needed for "Use sourceformat free/variable".